src/Payments/GPay.res

```diff
@@ -68,7 +74,7 @@ let default = (props: props) => {
     )
   }
 
-  React.useEffect0(() => {
+  React.useEffect(() => {
```

Comment:
Why is this change made? There is only an event listener inside this useEffect

Reply:
Added this while debugging.. will revert